client/scss/layouts/_account.scss

```
@@ -1,32 +1,22 @@
 .avatar-panel {
-  margin-top: 20px;
-
   &__image {
-    float: left;
-    width: 16%;
-    margin-bottom: 20px;
+    // do not display the span that is a sibling of the input when the input is checked
```

```suggestion
        // do not display the user image if the clearing image is checked
```

Try to make comments more related to why not what, reading the CSS tells you the what but it may not be clear about the why.

I have revised this comment as follows.